lib/truffle/java/java_ext/java_util_collection.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+java.util.Collection
+
+module ::Java::JavaUtil::Collection
+  def length
+    size
+  end
+
+  def include?(an_item)
+    self.contains(an_item)
+  end
```

Comment:
Ah, because you want `contains` to be a dynamic call and it's not defined yet in this `module`, I see.

I guess `contains` and Java calls are relying on `method_missing` and therefore `alias :include? :contains` will complain such as:
```
NameError: undefined method `contains' for class `T'
```
Is that the case? Or are Java-methods defined eagerly?